src/Sylius/Bundle/AdminBundle/Resources/views/Channel/_form_content.html.twig

```diff
@@ -0,0 +1,93 @@
+{{ form_errors(form) }}
+
+<div class="ui two column stackable grid">
+    {{ sylius_template_event('sylius.admin.channel.before_form', _context) }}
+    <div class="column">
+        <h4 class="ui top attached large header">{{ 'sylius.ui.general'|trans }}</h4>
+        <div class="ui attached segment">
+            {{ form_errors(form) }}
+            <div class="fields">
+                <div class="six wide field">
+                    {{ form_row(form.code) }}
+                </div>
+                <div class="seven wide field">
+                    {{ form_row(form.name) }}
+                </div>
+                <div class="three wide field">
+                    {{ form_row(form.color) }}
+                </div>
+            </div>
+            <div class="ui hidden divider"></div>
+            {{ form_row(form.enabled) }}
+        </div>
+        <div class="ui attached segment">
+            <div class="field">
+                {{ form_label(form.hostname) }}
+                <div class="ui labeled input">
+                    <div class="ui label">https://</div>
+                    {{ form_widget(form.hostname) }}
+                </div>
+                {{ form_errors(form.hostname) }}
+            </div>
+            {{ form_row(form.contactEmail) }}
+            {{ form_row(form.contactPhoneNumber) }}
+            {{ form_row(form.description, {'attr': {'rows' : '3'}}) }}
+        </div>
+        <div class="ui attached segment">
+            {{ form_row(form.countries) }}
+        </div>
+        <div class="ui hidden divider"></div>
+        <h4 class="ui top attached large header">{{ 'sylius.ui.money'|trans }}</h4>
+        <div class="ui attached segment">
+            <div class="two fields">
+                {{ form_row(form.baseCurrency) }}
+                {{ form_row(form.currencies) }}
+            </div>
+        </div>
+        <div class="ui attached segment">
+            {{ form_row(form.defaultTaxZone) }}
+            {{ form_row(form.taxCalculationStrategy) }}
+        </div>
+
+        {{ sylius_template_event('sylius.admin.channel.after_first_column_form', _context) }}
```

Comment:
What about creating `sylius.admin.channel.form_columns` event, with two blocks (for each column each)? Therefore, we could add anything before or after any of the columns. But that would not resolve the issue you try to address here. So it is just an improvement.

Looking at this template, I would like to start coining the rule of thumb: "If you want to create the event with "after" or "before" keyword in name, just create a block inside of event without these words" - do you think that such rule makes sense? 

```suggestion
{{ sylius_template_event('sylius.admin.channel.first_form_column', _context) }} 
```
Inside such an event, we should put the content from this column in one block. We won't need "after" or "before" events. New blocks would do the trick. What do you think?

Reply:
as i get it right, this would allow devs to overwrite first column, but wont allow to just add one field after the column is created, right?

Reply:
I would be for defining blocks per column (as Łukasz suggested) and also defining in this blocks other events for column content, only in that way, it will be truly customizable